bindings/rust/bench/benches/handshake.rs

```diff
@@ -35,13 +39,24 @@ pub fn bench_handshake_key_exchange(c: &mut Criterion) {
         });
     }
 
-    for ec_group in [SECP256R1, X25519] {
-        let mut bench_group = c.benchmark_group(format!("handshake-{:?}", ec_group));
-        bench_handshake_for_library::<S2NHarness>(&mut bench_group, &ec_group);
-        bench_handshake_for_library::<RustlsHarness>(&mut bench_group, &ec_group);
-        bench_handshake_for_library::<OpenSslHarness>(&mut bench_group, &ec_group);
+    for handshake_type in [NoClientAuth, ClientAuth] {
+        for ec_group in [SECP256R1, X25519] {
+            let mut bench_group =
```

Comment:
formatting: It looks like the indenting is a little funky here. I'm growing increasingly suspicious that our CI isn't running on this crate 😬

Reply:
Hmm. I just don't like the way that `fmt` put the entire next expression on a single line with an additional level of indent. But we should just stick with the _one true way_.